topic-controller/src/test/java/io/strimzi/controller/topic/ControllerTest.java

```diff
@@ -683,17 +680,17 @@ public void testReconcile_withCm_withKafka_withPrivate_3WayMerge(TestContext con
     // + error cases
 
     private void configMapRemoved(TestContext context, Exception deleteTopicException, Exception storeException) {
-        Topic kubeTopic = new Topic.Builder(topicName.toString(), 10, (short)2, map("cleanup.policy", "bar")).build();
+        Topic kubeTopic = new Topic.Builder(topicName.toString(), 10, (short) 2, map("cleanup.policy", "bar")).build();
         Topic kafkaTopic = kubeTopic;
         Topic privateTopic = kubeTopic;
 
         mockKafka.setCreateTopicResponse(topicName.toString(), null)
-                .createTopic(kafkaTopic, ar -> {});
+                .createTopic(kafkaTopic, ar -> { });
```

Comment:
This is something I can't understand ... but maybe it's related to having spaces when operators are used which I really like :-)